userscripts/commit-ref.user.js

```diff
@@ -1,8 +1,9 @@
 // ==UserScript==
 // @name         GitWeb: copy commit reference
-// @version      2
+// @version      3
 // @author       Andrei Rybak
 // @description  Adds a "Copy commit reference" button to every commit page on GitWeb websites.
+// @license      AGPL-3.0-only
```

Comment:
Greasemonkey gave a warning with `@license`. It has no effect on running the script, but it doesn't seem like Greasemonkey actually parses the `@license` line.